Formula/d/docker-buildx.rb

```diff
@@ -32,11 +33,18 @@ def install
     generate_completions_from_executable(bin/"docker-buildx", "completion")
   end
 
+  def post_install
```

Comment:
We could probably replace this `post_install` entirely by doing something like
```ruby
(lib/"docker/cli-plugins").install_symlink bin/"docker-buildx"
```
Any reason why we're not doing that?

Reply:
That's indeed simpler; I updated the changes. PHAL